Repository/ServiceEntityRepository.php

```diff
@@ -22,7 +22,7 @@
  *     }
  * }
  *
- * @template T
+ * @template T of object
```

Comment:
@derrabus This change is triggering a `MoreSpecificReturnType` and `LessSpecificReturnStatement` issue for me while using ORM 2.12.2 and Persistance 2.5.3, while it was working properly before.

Reply:
triggerring them where ?